src/plugins/advanced_settings/public/management_app/components/advanced_settings_voice_announcement/advanced_settings_voice_announcement.tsx

```diff
@@ -89,7 +97,7 @@ export class AdvancedSettingsVoiceAnnouncement extends Component<Props> {
               />
             )}
           </EuiDelayRender>
-        </div>
+        </>
```

Comment:
```suggestion
        </div>
```

Assuming that the self-closing `div` above was a mistake, this should be reverted as well.

Reply:
addressed it